frontend/views/pages/PayGroup.vue

```diff
@@ -318,65 +515,196 @@ export default {
 <style lang="scss" scoped>
 @import "@assets/style/_variables.scss";
 
+.p-descritpion {
+  margin-top: -1.8rem;
```

Comment:
we should aim for values that match the DS unless extremely needed. I think here it should be -1.5rem.